addons/sourcemod/scripting/tfgo.sp

```diff
@@ -278,6 +280,19 @@ public void OnClientDisconnect(int client)
 	}
 }
 
+public Action OnTakeDamage(int &attacker, int &inflictor, float &damage, int &damagetype, int &weapon, float damageForce[3], float damagePosition[3], int damagecustom)
+{
+	if (tfgo_all_weapons_can_headshot.BoolValue)
+	{
+		damagetype &= DMG_USE_HITLOCATIONS;
```

Comment:
Should it be `damagetype |= DMG_USE_HITLOCATIONS` to add flag? Never seen just `&=` used